### ISSUE TITLE
Check docs are up to date

### ISSUE THREAD

**Issue by ErikSchierboom** [exercism/shen#55]
Please check if your documentation files are still up-to-date.

The key documentation files to check are:

- `docs/ABOUT.md`
- `docs/INSTALLATION.md`
- `docs/LEARNING.md`
- `docs/RESOURCES.md`
- `docs/TESTS.md`
- `exercises/shared/.docs/help.md`
- `exercises/shared/.docs/tests.md`

There might be more.

## Link check report

To help identify invalid links, we've automatically checked the links of _all_ `*.md` files in this repo.
This is the report of that check:

<pre style="color:#bbb;white-space:pre-wrap;word-wrap:break-word;overflow-wrap:break-word">📝 Summary
---------------------
🔍 Total............0
✅ Successful.......0
⏳ Timeouts.........0
🔀 Redirected.......0
👻 Excluded.........0
🚫 Errors...........0

</pre>
## Tracking

https://github.com/exercism/v3-launch/issues/54

**Comment by girijakar** [comment]
Hi, I would like to work on this.

**Comment by jaccarmac** [comment]
As you can see, I haven't been working on this very hard. In fact, I haven't done much Shen hacking at all, though I'm interested in getting back to it. Do you have a particular interest in the docs?

**Comment by girijakar** [comment]
NO, not any particular interest. I wanted to contribute to exercism. I found this issue and thought of working on it :)
